app/public/cantusdata/debug_settings.py

```diff
@@ -0,0 +1,22 @@
+"""
+Overwrites some main settings that are used in development.
+"""
+
+from .settings import *
+
+DEBUG = is_development
+
+DEBUG_TOOLBAR_CONFIG = {
```

Comment:
You might want to add `"DISABLE_PANELS"` here and disable all the panels by default, which will save a lot of loading time. Then you can add them back as needed when you're debugging.